metabase/metabase
# fix breakout charts with nulls and empty strings

Closes https://github.com/metabase/metabase/issues/41610

### Description

Fix charts crash when breakout column has both `null` and `""` values. Previously existing behavior is far from being ideal, but fixing it is out of the scope of this project. Basically, we show both breakout series as `(empty)` and they share the same series settings values. At least, this shows the creator that query should be tweaked to either filter out some of these breakout values or group them together.

### How to verify

Create a native query
```sql
select 1 x, '' b, 10 y
union all select 1, null, 10
```
Ensure it shows two different series

### Demo

<img width="1098" alt="Screenshot 2024-04-19 at 6 32 29 PM" src="https://github.com/metabase/metabase/assets/14301985/a7263e06-ae19-4141-9dec-6ae57476b1a5">

### Checklist
